### PR TITLE
images: Simplify rhel.setup, add rhel-9-0

### PR DESCRIPTION
**Pull request by martinpitt**
Also rebuilding the rhel-8* images to ensure that they did not regress.

 * [x] image-refresh rhel-8-4
 * [x] image-refresh rhel-8-5
 * [x] image-refresh rhel-9-0
 * [x] builds on top of #1856

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 3-cockpit-7.
Log: https://logs.cockpit-project.org/logs/image-refresh-1859-20210329-062323/

**Comment by cockpituous** [comment]
image-refresh rhel-8-4 done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel-8-4-20210329-064141

**Comment by cockpituous** [comment]
image-refresh in progress on 3-ci-srv-06.
Log: https://logs.cockpit-project.org/logs/image-refresh-1859-20210329-064242/

**Comment by cockpituous** [comment]
image-refresh rhel-8-5 done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel-8-5-20210329-070404

**Comment by cockpituous** [comment]
image-refresh in progress on 3-ci-srv-04.
Log: https://logs.cockpit-project.org/logs/image-refresh-1859-20210329-070404/

**Comment by cockpituous** [comment]
image-refresh rhel-9-0 done: https://github.com/cockpit-project/bots/commits/image-refresh-rhel-9-0-20210329-071616

**Comment by martinpitt** [comment]
#1859 landed, so rebasing. The [lorax PR that fixes the test](https://github.com/weldr/lorax/pull/1130) also just landed, so let's just re-run everything to get something nice and green.

**Comment by martinpitt** [comment]
The new rhel-9-0 image is exercised in https://github.com/cockpit-project/cockpit/pull/15614 . I am still working on adjusting the tests, but that doesn't need to block landing this.

File: images/scripts/rhel.setup
Comment (marusak) [inline images/scripts/rhel.setup:267]: What is this option in mock? From doc I see:
`config_opts['root'] = 'name-of-yum-build-dir'` which is not that clear as one would hope for. But if I understand this correctly this is just to name your build location?

Comment (martinpitt) [reply images/scripts/rhel.setup:267]: Right, I suppose with koji etc. it will appear in some logs and/or URL paths, but for our purposes we don't care.

**Review by marusak — APPROVED** [review]
Thanks!

While I agree that we don't need to block this until rhel-9 passes all tests, but we should not merge this if this new image has some obvious issues that will need to adjust the image straight away (like missing packages).

So good for me but lets make sure that rhel-9 in https://github.com/cockpit-project/cockpit/pull/15614 is reasonably green.

**Comment by martinpitt** [comment]
@marusak: I'm sure we'll need some more modifications -- at least [more naughties](https://logs.cockpit-project.org/logs/pull-15614-20210329-111030-5294df25-rhel-9-0-bots-1859/log.html#73-2) like for that metrics failure. It also needs another rebuild for the now [missing dhclient package](https://logs.cockpit-project.org/logs/pull-15614-20210329-111030-5294df25-rhel-9-0-bots-1859/log.html#85-2).

The painful bit here is the rebuild and re-test of the rhel-8* images, and blocking them for possibly several days until I/we wade through all the 9.0 regressions. So if possible I'd like to do the further fine-tuning in a follow-up.

**Pull request merged**
